src/forms/ErrorList.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+export const ERROR_CLASSNAME = 'text--error text--small';
+
+const ErrorList = (props) => {
+	const {
+		errorId,
+		errors,
+		...other
+	} = props;
+
+	const errorList = Array.isArray(errors) ? errors : [errors];
```

Comment:
IMO `errors` should always be an array - no need to be super adaptable here, and it's a plural prop so sending a single string would be a little confusing to future maintainers